app/controllers/sessions_controller.rb

```diff
@@ -1,20 +1,116 @@
 class SessionsController < ApplicationController
-  def new; end
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i(index edit update destroy)
```
sửa tương tự những chỗ khác